xmtp_mls/src/groups/message_history.rs

```diff
@@ -199,7 +212,17 @@ where
     }
 }
 
-#[allow(dead_code)]
+fn write_groups_file(groups: Vec<StoredGroup>) -> Result<PathBuf, MessageHistoryError> {
+    let tmp_dir = std::env::temp_dir();
+    let file_path = tmp_dir.join("groups.jsonl");
```

Comment:
Might be good to make "groups.jsonl" a const maybe

Reply:
I have a follow up PR that I'll put up soon™ that refactors this to make it generic.  i.e. this is going away.